docs/src/pages/components/links/ButtonLink.js

```diff
@@ -3,7 +3,7 @@
 import React from 'react';
 import Link from '@material-ui/core/Link';
 
-export default function ButtonLink() {
+function ButtonLink() {
```

Comment:
I think it's fine to leave the immediate `export default`

Reply:
I think I remember reading we are starting to prefer using the immediate export when a component doesn't use any HOCs cc @oliviertassinari

Reply:
Yes, please use the direct export. I want to propagate this change to all the demos of the documentation. It was initially done the other way around to make it easy to introduce a new HOC. It's no longer something we need to optimize for. Instead, we can remove the name duplication and shorten the demos.

Reply:
@oliviertassinari #16005